externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkTBinaryFrontendService.scala

```diff
@@ -37,7 +37,7 @@ class SparkTBinaryFrontendService(
   private lazy val sc = be.asInstanceOf[SparkSQLBackendService].sparkSession.sparkContext
 
   override def RenewDelegationToken(req: TRenewDelegationTokenReq): TRenewDelegationTokenResp = {
-    debug(req.toString)
+    debug(s"${req.toString}")
```

Comment:
@cxzl25 , no need to change. Please revert this line.

Reply:
Done.
But shouldn't the code avoid splicing strings in advance when outputting the log?

Reply:
the input of `debug` is a by-name parameter so it promises the evaluation is lazy

Reply:
Thanks all